### PR TITLE
feat(code): Create `Code` component

### PR DESCRIPTION
**Pull request by guscsales**
## Description

- Creation of `Code` component and package.
- Small fixes on root README and container README files

## Screenshots
image

### CODE REVIEW

File: packages/code/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@react-email/code",
+  "version": "0.0.1",
+  "description": "A block of code to be rendered in your email.",
```
Comment (bukinoshita) [inline packages/code/package.json:4]:
```suggestion
  "description": "A block of code to be rendered in your email",
```

**Comment by bukinoshita** [comment]
Can we rename the PR title to:

```diff
- feat(code): create Code component
+ feat(code): Create Code component
```

**Comment by guscsales** [comment]
@bukinoshita both done

**Comment by zenorocha** [comment]
That's awesome!! Thanks @guscsales 🎉🎉🎉

**Pull request merged**